flatland/envs/persistence.py

```diff
@@ -200,6 +199,15 @@ def set_full_state(cls, env, env_dict):
         env.rail.width = env.width
         env.dones = dict.fromkeys(list(range(env.get_num_agents())) + ["__all__"], False)
 
+        # TODO merge with https://github.com/flatland-association/flatland-rl/pull/97/files
```

Comment:
TODO merge with https://github.com/flatland-association/flatland-rl/pull/97/files - keep for now?